tests/conftest.py

```diff
@@ -51,27 +56,40 @@ def local():
 @pytest.fixture()
 def sync():
     "Synchronous dask (not dask.distributed) executor"
-    yield LocalDaskExecutor()
+    yield LocalDaskExecutor(scheduler="sync")
+
+
+@pytest.fixture()
+def mproc_local():
+    "Multiprocessing executor using local dask (not distributed cluster)"
+    yield LocalDaskExecutor(scheduler="processes")
 
 
 @pytest.fixture(scope="session")
 def mproc():
-    "Multi-processing executor"
+    "Multi-processing executor using dask distributed"
     with Client(
         processes=True, scheduler_port=0, dashboard_address=":0", n_workers=2
     ) as client:
-        yield DaskExecutor(client.scheduler.address)
+        executor = DaskExecutor(client.scheduler.address)
+        # Since the cluster can't be inspected until the client is started we can't
+        # know if processes are being used before `flow.run` is called so we patch
+        # the class with an indicator
+        executor.__processes = True
```

Comment:
I don't love attaching an attribute to the executor but don't see another way to grab this config value during the test

Reply:
Yeah, I'd prefer to avoid this.